cdap-spark-python/src/main/resources/cdap/pyspark/log.py

```diff
@@ -24,4 +25,4 @@ def __init__(self):
     Handler.__init__(self)
 
   def emit(self, event):
-    print("Event", dir(event))
+    print(("Event", dir(event)))
```

Comment:
Does the extra parenthesis mean this is printing a tuple instead of 2 separate entries?

Reply:
I am not an expert, but it looks like it's part of python3 conversion, see https://stackoverflow.com/questions/55559825/how-to-fix-print-double-parentheses-after-2to3-conversion. @sagarrbatchu, could you please comment?

Reply:
that was part of the python2 to 3 upgrade using `futurize` . The double parenthesis is to indicate in python2 it was already printing as a tuple so this is to maintain that in python3